pubsub/src/main/java/com/github/twitch4j/pubsub/domain/PredictionResult.java

```diff
@@ -0,0 +1,26 @@
+package com.github.twitch4j.pubsub.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Data
+@Setter(AccessLevel.NONE)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PredictionResult {
+
+    private String type; // e.g. WIN or LOSE
+
+    private Integer pointsWon; // can be null
```

Comment:
Lets use the nullable annotation here too

Reply:
I'd recommend use [JRS-305](https://mvnrepository.com/artifact/com.google.code.findbugs/jsr305/3.0.2) for all repo. Will be better markable as nullable.

Reply:
We haven't really used the nullable/notnull annotation before so we're free to pick i think, the auto generated code in graphql uses the jetbrains annotation dependency.

Is there a reason why we should use JSR305 over the jetbrains annotation dependency?

Reply:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360003485599/comments/360000474759 - up to you